tensorflow_addons/image/testing_imageops.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
<p>I'd support we could use <code>tf.io.read_file</code> directly so that we do not need to access the private API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>